### DISCUSSION TITLE
4.13.4 Safe?

### DISCUSSION THREAD

**Discussion by tasi333** [alandtse/alexa_media_player#2600] [Polls]
For those that have installed 4.13.4, is it generally safe, no memory issues or crashing HA?

I'm still on 4.13.1 and it works, so I'm currently in the mindset of "if ain't broke, don't fix it".

But, if no-one is seeing any problems with it, I'll give it a shot.

Thanks

**Comment by stamandr** [comment]
I'm going to sit on the fence and watch others responses for a few days before I update....

**Reply by tasi333** [reply]
Same :)

**Comment by jlr2000** [comment]
Updated with zero issues.  Thanks to all who resolved this!!

**Comment by tasi333** [comment]
Ok, I took the plunge, so far so good, no issues.

**Comment by Sbechaalany** [comment]
working good with no error, i installed it since 2 days and everything is fine
